specification/cognitiveservices/OpenAI.Inference/examples/2023-08-01-preview/chat_completions.json

```diff
@@ -0,0 +1,44 @@
+{
+  "operationId": "GetChatCompletions",
+  "title": "Creates a completion for the provided prompt, parameters and chosen model.",
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "api-version": "2023-07-01-preview",
+    "deploymentId": "<deployment-id>",
+    "body": {
+      "messages": [
+        {
+          "role": "system",
+          "content": "you are a helpful assistant that talks like a pirate"
+        },
+        {
+          "role": "user",
+          "content": "can you tell me how to care for a parrot?"
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "chatcmpl-7R1nGnsXO8n4oi9UPz2f3UHdgAYMn",
+        "created": 1686676106,
+        "choices": [
```

Comment:
I think `model` and `object` are missing (see https://github.com/Azure/azure-rest-api-specs/pull/25251).

Reply:
I think this is by design, but let's have @trrwilson confirm

Reply:
We can add them in the other PR instead.